src/conduits/java/com/enderio/conduits/common/blockentity/ConduitBlockEntity.java

```diff
@@ -508,8 +509,8 @@ public boolean isInitialized() {
 
         public UpdateState next() {
             return switch (this) {
-                case NONE, INITIALIZED -> NONE;
-                case NEXT, MEXT_INITIALIZED -> INITIALIZED;
+                case NONE, INITIALIZED, MEXT_INITIALIZED -> NONE;
```

Comment:
I'm not sure about this agnor. This is the issue since it loops, and with this change, it does seem to update 2 times, but part of me also thinks this is wrong.

Reply:
some of the issues are fixed, but this change is wrong, now the me conduits don't connect anymore when a ae2 block is placed afterwards next to the conduit